referrer-policy/no-referrer-policy/no-referrer-policy.subresource.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>This page reports back it's referrer URL to the parent frame</title>
+    <!-- Common global functions for referrer-policy tests. -->
+    <script src="../generic/common.js"></script>
+  </head>
+  <body style="background:#f4f4f4;">
+    <h2>This page reports back it's referrer URL to the parent frame.</h2>
+
+    <script>
+      // Display this page's location (iframe page - subresource to parent).
+      appendPageInfo("Child frame");
+
+      var pageInfo = {
+          location:document.location.toString(),
+          referrer:document.referrer
```

Comment:
Note that this test only verifies the JavaScript interface. You could convert it to a Python file which outputs the HTTP header as well (e.g. `print "<script>var httpReferer = '%s';</script>" % request.headers["referer"]` (see something like https://github.com/w3c/web-platform-tests/blob/master/eventsource/resources/cors-cookie.py for something vaguely similar).